apps/src/templates/studioHomepages/TwoColumnActionBlock.jsx

```diff
@@ -169,20 +169,57 @@ export class AdministratorResourcesActionBlock extends Component {
 }
 
 export class SpecialAnnouncementActionBlock extends Component {
+  static propTypes = {
+    hocLaunch: PropTypes.string
+  };
 
   render() {
-    return (
-      <TwoColumnActionBlock
-        imageUrl={pegasus('/shared/images/fill-540x289/teacher-announcement/hoc2018.jpg')}
-        subHeading={i18n.specialAnnouncementHeadingHoc2018()}
-        description={i18n.specialAnnouncementDescriptionHoc2018()}
-        buttons={[
-          {
-            url: 'https://hourofcode.com/#join',
-            text: i18n.signUpButton()
-          },
-        ]}
-      />
-    );
+    if (this.props.hocLaunch === "dance") {
```

Comment:
I don't know if it matters but most of our components handle the conditional rendering all within one return. 

`return ( 
{this.props.hocLaunch === "dance" && <TwoColumnActionBlock with one set of props}
{this.props.hocLaunch === "mc" && <TwoColumnActionBlock with another set of props}`

Reply:
I did ponder that, but figured it was a bit odd to launch right into conditionals in JSX when we could hoist them to the surrounding JS.  Most of this code is temporary anyway, so I also figured it would be less churn when we go back to a single piece of JSX in this function.